mp/src/game/client/c_baseviewmodel.cpp

```diff
@@ -40,6 +40,8 @@ MAKE_TOGGLE_CONVAR_CV(cl_righthand, "1", FCVAR_ARCHIVE, "Use right-handed view m
     return true;
 });
 
+MAKE_CONVAR(r_viewmodel_opacity, "255", FCVAR_ARCHIVE, "Set the opacity of view models. MIN = 10, MAX = 255.\n", 10.0f, 255.0f);
```

Comment:
Since this isn't a color component you can just make this a scale from 0.01 to 1 and remove the division done at setting the blend

Reply:
Good point, latest commit changes this